src/models/artboard/README.md

```diff
@@ -0,0 +1,66 @@
+# ArtBoard Model
+
+ArtBoard Model is responsible for storing the structure of the class and its parameter used for artBoardDraw funtionality.These classes are called by the `ArtBoardSketch.tsx` and `ArtBoardTurtle.tsx` There are majorly 4 different classes.
+
+<ul>
+<li> ArtBoardDraw.ts
+<li> ArBoard.ts
+<li> ArBoardManager.ts
+<li> Turtle.ts
+</ul>
+
+---
+
+&nbsp;
+
+## ArBoardDraw.ts
+
+Responsible for storing the thickness and the color of the line drawn on the artBoard. Each instance of `artBoardSketch.tsx` calls artBoardDraw constructor with color (RGB format) as the parameter.
```

Comment:
Responsible for storing the attributes (thickness and color) of the line drawn on the artBoard. Each instance of `artBoardSketch.tsx` calls artBoardDraw constructor with color (RGB format) as the parameter.